common/presenters/allocations-to-table-component.js

```diff
@@ -4,6 +4,13 @@ const componentService = require('../services/component')
 
 const tablePresenters = require('./table')
 
+function showProgress(data, showRemaining) {
```

Comment:
This function could do with some tests around it.

Also, this function and the ones below it look like they're pure functions, so should be easy to extract out and test individually.

Reply:
Covered the scenario. I would usually agree with you about how it should be done, but presenters take input and spit back output and I'm following the way its been done.